src/Core/src/Platform/Windows/WindowRootView.cs

```diff
@@ -78,58 +78,52 @@ public WindowRootView()
 				if (_appTitleBar != null)
 					return _appTitleBar;
 
-				if (AppTitleBarContentControl == null)
+				if (AppTitleBarContentControl is null)
 					return null;
 
 				var cp = AppTitleBarContentControl.GetFirstDescendant<ContentPresenter>();
 
-				if (cp == null)
+				if (cp is null)
 					return null;
 
 				_appTitleBar = cp.GetFirstDescendant<FrameworkElement>();
 
-				if (_appTitleBar != null)
-				{
-					OnAppTitleBarChanged?.Invoke(this, EventArgs.Empty);
-				}
-				else
+				if (_appTitleBar is null)
 				{
 					_appTitleBar = cp;
-					OnAppTitleBarChanged?.Invoke(this, EventArgs.Empty);
 				}
 
+				if (_appTitleBar is Border border && _setTitleBarBackgroundToTransparent)
+					border.Background = null;
```

Comment:
This looks interesting... Is this a maui border? What happens if there is a grid or some other layout - or a Frame? 

What is the reason for this? The dragging? Should we rather check for wrapper views or layouts instead?

Reply:
One thing to keep in mind here is that this PR is the first step to resolving this issue https://github.com/dotnet/maui/issues/13023. 

Once I get that implemented, the `TitleBar` will just be an xplat component that users can define themselves with a `ControlTemplate` or similar. At that point we will just be setting `Transparent` on our own controls or if a user wants to define their own template then it's on them to manage this.  I debated just converting this to an xplat template for now on this PR but opted for just doing it all as one PR.

This border is currently all defined inside Windows Styles
https://github.com/dotnet/maui/blob/main/src/Core/src/Platform/Windows/Styles/WindowRootViewStyle.xaml#L17